To-do-List.js

```diff
@@ -0,0 +1,232 @@
+let items = [];
+let itemsCopy = [];
+let text, j;
+
+let addForm = () => {
+    document.getElementById('form').style.visibility = 'visible';
+};
+
+let addItem = () => {
+    var appendItems = document.getElementById('appendItem');
+    appendItems.innerHTML = '';
+
+    document.getElementById('form').style.visibility = 'hidden';
+    text = document.getElementById('item').value;
+    items.push({
+        text: text,
+        status: "unchecked"
+    });
+
+    itemsCopy = items.slice();    
```

Comment:
Remove slice not needed to copy the same array into another one.

Reply:
Yes. I forgot it is now array of objects.